django/core/management/commands/shell.py

```diff
@@ -118,11 +120,52 @@ def python(self, options):
         # Start the interactive interpreter.
         code.interact(local=imported_objects)
 
-    def get_and_report_namespace(self, **options):
+    def get_auto_imports(self):
+        """Return a sequence of import paths for objects to be auto-imported.
```

Comment:
(nit) Although I think you've done this to help folks override this correctly, I think if we have a doc string, we should mention that we are importing the models and that the example should include some models (like User)